templates/base.html

```diff
@@ -9,6 +10,20 @@
     <link rel="stylesheet" type="text/css" href="{% static "jquery.datetimepicker.css" %}" />
     <link rel="stylesheet" href="//ajax.googleapis.com/ajax/libs/jqueryui/1.11.1/themes/smoothness/jquery-ui.css" />
 
+    {% load raven %}
+    <script src="https://cdn.ravenjs.com/3.17.0/raven.min.js" crossorigin="anonymous"></script>
+    <script>
+        Raven.config('{% sentry_public_dsn %}', {
+            release: '{% settings_value "RAVEN_RELEASE" %}',
+            environment: '{% settings_value "RAVEN_ENVIRONMENT" %}'
+        }).install();
+
+        // TODO: remove this
```

Comment:
Leaving this in for first merge?

Reply:
Nah, its there for testing because as of right now this still doesn't work, Sentry returns a `400` when trying to send payloads.